CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [3.0.0-rc3] - 2020-09-11
```

Comment:
I think we were intending to collapse all the RCs into 3.0.0 - can we delete the 3.0.0-rc2 lines?